canonical-web-and-design/ubuntu.com
# update pricing/devices copy

## Done

- Refreshed the content on /pricing/devices according to the [copy doc](https://docs.google.com/document/d/1uiKNaT59tk2rvjuM24vpqVblcNO90PmE4wUlq9A9Rsk/edit#)

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/pricing/devices
    - Be sure to test on mobile, tablet and desktop screen sizes
- See that the content matches the copy doc

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/5081
